packages/dev/core/src/ShadersWGSL/iblGenerateVoxelMip.fragment.fx

```diff
@@ -0,0 +1,30 @@
+varying vUV: vec2f;
+
+var srcMip: texture_3d<f32>;
+
+uniform layerNum: i32;
+
+@fragment
+fn main(input: FragmentInputs) -> FragmentOutputs {
+  var Coords = vec3i(2) * vec3i(i32(input.position.x), i32(input.position.y), uniforms.layerNum);
```

Comment:
shader preprocessor can't handle this expression see [here](https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Engines/WebGPU/webgpuShaderProcessorsWGSL.ts#L280)
```suggestion
  var Coords = vec3i(2) * vec3i(fragmentInputs.position.xy), uniforms.layerNum);
```